lib/BloqadeKrylov/Project.toml

```diff
@@ -4,13 +4,17 @@ version = "0.1.7"
 
 [deps]
 Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
+Bloqade = "bd27d05e-4ce1-5e79-84dd-c5d7d508bbe1"
 BloqadeExpr = "bd27d05e-4ce1-5e79-84dd-c5d7d508abe2"
 BloqadeLattices = "bd27d05e-4ce1-5e79-84dd-c5d7d508bbe4"
 BloqadeWaveforms = "bd27d05e-4ce1-5e79-84dd-c5d7d508bbe7"
 Configurations = "5218b696-f38b-4ac9-8b61-a12ec717816d"
 ExponentialUtilities = "d4d017d3-3776-5f7e-afef-a10c40355c18"
+GaussQuadrature = "d54b0c1a-921d-58e0-8e36-89d8069c0969"
+LaTeXStrings = "b964fa9f-0449-5b57-a5c2-d3ea65f4040f"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 ProgressLogging = "33c8b6b6-d38a-422a-b730-caa89a2f386c"
+Yao = "5872b779-8223-5990-8dd0-5abbb0748c8c"
 YaoArrayRegister = "e600142f-9330-5003-8abb-0ebd767abc51"
 YaoSubspaceArrayReg = "bd27d05e-4ce1-5e79-84dd-c5d7d508ade2"
```

Comment:
How many of these new packages are required for running the benchmarks vs. just installing the package?

Reply:
GaussQuadrature is needed, I will remove the rest.